<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileProperties</class>
 <widget class="QDialog" name="FileProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>317</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>317</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Propiedades</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Resources/logo.png</normaloff>:/Resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>106</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>267</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Permisos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>305</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubicación:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>186</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha ultima modificación</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Indice:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>105</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Extensión:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>227</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>226</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Padre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>145</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha creación:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>225</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Propietario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExtension">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFechaCreacion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>160</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFechaMod">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>200</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtUbicacion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>320</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPermisos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>40</y>
     <width>280</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/barra.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>400</y>
     <width>280</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/barra.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>40</y>
     <width>2</width>
     <height>362</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/barra.png);
</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>40</y>
     <width>2</width>
     <height>362</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/barra.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nyala</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Información del archivo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtIndice">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>65</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>411</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/fondo.png);</string>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/Resources/db_comit.png</normaloff>:/Resources/db_comit.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>411</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/fondo.png);</string>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/Resources/button_delete_blue.png</normaloff>:/Resources/button_delete_blue.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btnNewFolder">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/fondo.png);
</string>
   </property>
   <property name="text">
    <string>Nueva Carpeta</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/Resources/Folder-2.png</normaloff>:/Resources/Folder-2.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>266</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Posición</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPosicion">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>280</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPadre">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTamano">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPropietario">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtIndice</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>btnNewFolder</tabstop>
  <tabstop>btnOpen</tabstop>
  <tabstop>txtExtension</tabstop>
  <tabstop>txtFechaCreacion</tabstop>
  <tabstop>txtFechaMod</tabstop>
  <tabstop>txtTamano</tabstop>
  <tabstop>txtPadre</tabstop>
  <tabstop>txtPropietario</tabstop>
  <tabstop>txtPosicion</tabstop>
  <tabstop>txtPermisos</tabstop>
  <tabstop>txtUbicacion</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
